nix/default.nix

```diff
@@ -358,7 +359,11 @@ let
         export E2E_EXTRA_BROWSER_ARGS="--disable-web-security"
 
         python -m http.server 4008 --directory ${bundledPursProject}/dist &
-        sleep 3 # Wait for it to start serving
+        until curl -S http://127.0.0.1:4008/index.html &>/dev/null; do
+          echo "Trying to connect to webserver...";
+          sleep 0.1;
```

Comment:
Seems aggressive if 3 was too slow

Reply:
Yep, but I don't want to wait 6 seconds for a 3.01 delay